templates/etc/systemd/system/gitlab-workhorse.service.j2

```diff
@@ -18,14 +11,21 @@ PartOf=gitlab.slice
 
 [Service]
 Slice=gitlab.slice
-Type=forking
+Type=simple
 User={{ gitlab_user }}
 WorkingDirectory={{ gitlab_app_root_path }}/gitlab-workhorse
 SyslogIdentifier=gitlab-workhorse
 PIDFile={{ gitlab_app_root_path }}/gitlab/tmp/pids/gitlab-workhorse.pid
 Environment="PATH=/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:{{ gitlab_app_root_path }}/gitlab-workhorse"
 
-ExecStart={{ gitlab_app_root_path }}/gitlab/bin/daemon_with_pidfile {{ gitlab_app_root_path }}/gitlab/tmp/pids/gitlab-workhorse.pid {{ gitlab_app_root_path }}/gitlab-workhorse/gitlab-workhorse -listenUmask 0 -listenNetwork unix -listenAddr {{ gitlab_app_root_path }}/gitlab/tmp/sockets/gitlab-workhorse.socket -authBackend http://127.0.0.1:8080 -authSocket {{ gitlab_app_root_path }}/gitlab/tmp/sockets/gitlab.socket -secretPath {{ gitlab_app_root_path }}/gitlab/.gitlab_workhorse_secret -documentRoot {{ gitlab_app_root_path }}/gitlab/public >> {{ gitlab_app_root_path }}/gitlab/log/gitlab-workhorse.log 2>&1
+ExecStart={{ gitlab_app_root_path }}/gitlab-workhorse/gitlab-workhorse \
+    -listenUmask 0 \
+    -listenNetwork unix \
+    -listenAddr {{ gitlab_app_root_path }}/gitlab/tmp/sockets/gitlab-workhorse.socket \
+    -authBackend http://127.0.0.1:8080 \
```

Comment:
This would be nice to have on a different port, since `8080` is already reserved as `http-alt`. Is this configurable within GitLab? Any chance to switch to a UNIX socket?